src/Featurizers/ShortGrainDropperFeaturizer.h

```diff
@@ -0,0 +1,255 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#pragma once
+
+#include "../Featurizer.h"
+#include "../Archive.h"
+
+namespace Microsoft {
+namespace Featurizer {
+namespace Featurizers {
+
+
+/////////////////////////////////////////////////////////////////////////
+///  \class         ShortGrainDropperTransformer
+///  \brief         Return true if a grain needs dropping
+///
+class ShortGrainDropperTransformer : public StandardTransformer<std::vector<std::string>, bool> {
+public:
+    // ----------------------------------------------------------------------
+    // |
+    // |  Public Types
+    // |
+    // ----------------------------------------------------------------------
+    using BaseType                          = StandardTransformer<std::vector<std::string>, bool>;
+    using GrainsSet                         = std::unordered_set<std::string>;
```

Comment:
><std::string [](start = 64, length = 12)

Should this be `std::vector<std::string>` given that a single grain may be comprised of multiple strings?